src/containers/Secrets/Secrets.js

```diff
@@ -154,11 +169,20 @@ export /* istanbul ignore next */ class Secrets extends Component {
           }
         });
       }
+      const secretSA = [];
```

Comment:
naming: let's expand this to `secretServiceAccounts` and similar with other variables below, avoids additional cognitive load interpreting 'sa'/'SA'